chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/SurveillanceLite.java

```diff
@@ -0,0 +1,122 @@
+package gov.healthit.chpl.domain;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import org.springframework.beans.BeanUtils;
+
+import gov.healthit.chpl.dto.SurveillanceLiteDTO;
+import gov.healthit.chpl.util.Util;
+
+public class SurveillanceLite implements Serializable {
+    private static final long serialVersionUID = 3750079664886758825L;
+
+    private Long id;
+    private String friendlyId;
+    private Long certifiedProductId;
+    private Date startDate;
+    private Date endDate;
+    private Long surveillanceTypeId;
+    private SurveillanceType surveillanceType;
+    private Integer numRandomizedSites;
+    private Long userPermissionId;
+    private String chplProductNumber;
+
+    public SurveillanceLite() {
```

Comment:
We have a "CertifiedProductBasic" thing, I think, that's the Listing without the criteria/cqms/surveillance (IIRC). Would it make sense to keep a similar naming convention for a "SurveillanceBasic" object?